### PR TITLE
Add meeting minutes for 2015-03-19

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: doc/about/core-team/meetings/2015-03-19/minutes.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+how we can maintain support for older OS levels might might affect this.  
+
+#### ICU crash - Issue 9361
+Fix for this is already integrated in a later V8 version.  Steven Loomis is 
+looking at this and there is a pull request ready to backport the existing fix.  
+There was some discussion about the best approach and inthe end we agreed
+the existing fix is appropriate and we should just backport it to 0.12.2
+
+#### Testing release candidates
+Julien is talking wtih some groups about doing some pre-release testing on
```
Comment (lucamaraschi) [inline doc/about/core-team/meetings/2015-03-19/minutes.md]: typo: wtih should be **with**

**Comment by mhdawson** [comment]
fixed typo

**Comment by misterdjules** [comment]
LGTM with a few minor nits that I'll fix when landing it. Thank you again @mdawsonibm!

**Comment by misterdjules** [comment]
Landed in 75b309f199ea2c46b64b46323318a6d5f29a9c3e.

**Pull request closed without merging**